### PR TITLE
[trivial] Improved docs for std.conv.parse (part 4)

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

**Comment by qznc** [comment]
LGTM

File: std/conv.d [hunk 1]
```diff
@@ -3275,7 +3275,20 @@ Target parse(Target, Source)(ref Source s)
     assert(b == true);
 }
 
-// input range to null literal conversions
```
Comment (wilzbach) [inline std/conv.d]: It seems like this function wasn't exposed to the user before. Is there a good use case where a user might need this explicit documentation?

Comment (JackStouffer) [inline std/conv.d]: No public function should be undocumented. If the user is supposed to be able to use it, then they should know about it.

**Comment by JackStouffer** [comment]
Ping

**Comment by burner** [comment]
Auto-merge toggled on

**Pull request merged**